### PR TITLE
CI: skip Doxygen builds on forks

### PR DESCRIPTION
**Pull request by chipitsine**
:cl:
Skip Doxygen build on forks
/:cl:

:cl: Categories
CI
/:cl:

[Why]: small improvement

### CODE REVIEW

File: .github/workflows/ci-suite.yml [hunk 1]
```diff
@@ -184,7 +184,7 @@ jobs:
         doxyfile-path: 'docs/Doxyfile'
 
     - name: gh-pages push
-      if: github.event_name == 'push' && github.event.ref == 'refs/heads/dev'
+      if: github.event_name == 'push' && github.event.ref == 'refs/heads/dev' && env.TGS_RELEASE_NOTES_TOKEN != ''
```
Comment (Cyberboss) [inline .github/workflows/ci-suite.yml:187]: This variable still needs to be set at the job or step level with
```yml
env:
  TGS_RELEASE_NOTES_TOKEN: ${{ secrets.TGS_RELEASE_NOTES_TOKEN }}
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-suite.yml (hunk 1 above)
Comment (chipitsine) [reply .github/workflows/ci-suite.yml:189]: np

File: .github/workflows/ci-suite.yml [hunk 2]
```diff
@@ -160,6 +160,8 @@ jobs:
   dox-build:
     name: Build Doxygen Site
     runs-on: ubuntu-latest
+    env:
+      TGS_RELEASE_NOTES_TOKEN: ${{ secrets.TGS_RELEASE_NOTES_TOKEN }}
```
Comment (Cyberboss) [inline .github/workflows/ci-suite.yml:164]: Sorry, I obviously wasn't paying attention when I first looked.

We already have `TGS_RELEASE_NOTES_TOKEN` as an alias for `secrets.DEV_PUSH_TOKEN` defined in the global environment. You were right the first time.

Could you remove the environment definition please?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-suite.yml (hunk 2 above)
Comment (chipitsine) [reply .github/workflows/ci-suite.yml]: I discarded the latest commit

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**